persistence/src/commonMain/db_user/com/wire/kalium/persistence/Members.sq

```diff
@@ -20,8 +20,11 @@ DELETE FROM Member WHERE conversation = ? AND user = ?;
 selectAllMembersByConversation:
 SELECT * FROM Member WHERE conversation LIKE ('%' || :searchQuery || '%');
 
-selectAllConversationsByMember:
-SELECT * FROM Member WHERE user = ?;
+selectConversationByMember:
```

Comment:
would it make sense for this to be a view? It is used frequently or only on user interaction? (i.e. opening a profile) ? @yamilmedina

Reply:
In this case, this is only used on 2 user interactions only;

- when opening another user profile
- when clicking "open conversation". 

So I think we are good without views for this